packages/api-v2/test/auth/auth.e2e-spec.ts

```diff
@@ -0,0 +1,101 @@
+import { INestApplication } from "@nestjs/common";
+import { Test, TestingModule } from "@nestjs/testing";
+import { Student } from "../../src/student/entities/student.entity";
+import * as request from "supertest";
+import { Connection } from "typeorm";
+import { AppModule } from "../../src/app.module";
+
+const testUser = {
+  fullName: "Tester",
+  nuid: "000000000",
+  email: "test-auth@gmail.com",
+  password: "1234567890",
+  academicYear: 2019,
+  graduateYear: 2023,
+  catalogYear: 2019,
+  major: "Computer Science",
+};
```

Comment:
later on, we should setup factories (a la KOH) for testing. but can be done in later PR

see https://github.com/sandboxnu/office-hours/blob/master/packages/server/test/util/factories.ts for reference